src/Events/Services/OutboundService.cs

```diff
@@ -156,7 +160,7 @@ private Uri GetSourceFilter(Uri source)
             }
             else
             {
-                return null;
+                return source;
```

Comment:
not sure why you're changing the return type here? Do we need to update the flow for app events?

Reply:
GetSourceFilter is used as a cache key. I suggest refactoring to improve readability.